test/integration/assistant.v2.test.js

```diff
@@ -116,4 +116,19 @@ describe('assistant v2 integration', () => {
       done();
     });
   });
+
+  it('should listLogs @slow', done => {
```

Comment:
If this is one of the tests that requires a premium instance, you need to add a comment above describing that. Also, if I were to pull down this test and run it as is, with my credentials, my guess is that it would fail and I would be confused. So maybe we should comment it out entirely, or change the code to make it very clear that I would need new credentials to run it (like creating a specific instance of Assistant and using it only for this test, with the extra credentials needed).

Reply:
Makes sense, I will make that change along with the api updates